src/projection/node/create-projection-node.ts

```diff
@@ -356,6 +356,20 @@ export function createProjectionNode<I>({
                 this.root.registerSharedNode(layoutId, this)
             }
 
+            if (
+                this !== this.root &&
+                this.options.shouldMeasureScroll === undefined &&
+                this.instance instanceof Element
+            ) {
+                const computedStyle = getComputedStyle(this.instance)
+                if (
+                    computedStyle.overflowX === "scroll" ||
```

Comment:
This will trigger a synchronous style recalculation/layout https://gist.github.com/paulirish/5d52fb081b3570c81e3a#calling-getcomputedstyle

IMO it would be a reasonable API constraint to either 1) only check the React `style` prop and not support setting this via other CSS (preferable) or 2) make it an opt-in via a prop

Reply:
The first option is not enough to fix the issue as we do inject the CSS via a classname in FramerStudio. We can probably do both.

Reply:
If it's possible to also add it via style this would be much more performant.

Reply:
Introduced a new prop `shouldMeasureScroll` instead!

Reply:
Awesome!